### PR TITLE
feat: finish send token in wallet

### PR DESCRIPTION
**Pull request by angelia-yuqi-personal**
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/helios/215)

### CODE REVIEW

**Comment by angelia-yuqi-personal** [comment]
please pay close attention to `estimate-tx` package @yqrashawn

File: packages/popup/src/hooks/index.js [hunk 1]
```diff
@@ -64,33 +82,229 @@ export const useFontSize = (
...
+    Object.keys(tokensAmount)?.[0],
+    type,
+  ])
 
   if (loading) {
     return {loading}
   }
 
   if (error) {
+    console.log('error', error)
```
Comment (RickStormBringer) [inline packages/popup/src/hooks/index.js:123]: better to  remove console

File: packages/popup/src/hooks/useApi.js [hunk 2]
```diff
@@ -177,7 +177,7 @@ export const useBalance = (
 ) => {
   const {data: balance} = useRPC(
     address && isNumber(networkId) && isString(tokenContractAddress)
-      ? [WALLET_GET_BALANCE, address, networkId]
+      ? [WALLET_GET_BALANCE, address, networkId.toString, tokenContractAddress]
```
Comment (RickStormBringer) [inline packages/popup/src/hooks/useApi.js:180]: toString()

File: packages/popup/src/hooks/index.js (hunk 1 above)
Comment (angelia-yuqi-personal) [reply packages/popup/src/hooks/index.js:123]: it is used for debugging. when release I will remove the console.

File: packages/popup/src/components/DappFooter.js [hunk 3]
```diff
@@ -56,6 +58,10 @@ function DappFooter({
       case WALLET_SWITCH_CONFLUX_CHAIN:
         params.chainConfig = req.params
         break
+      case CFX_SEND_TRANSACTION:
+      case ETH_SEND_TRANSACTION:
+        params.tx = req.params
```
Comment (yqrashawn) [inline packages/popup/src/components/DappFooter.js:63]: so the `req` here is from `usePendingAuthReq`
but user can change the tx data with the send tx UI
is this `req` the user-altered one?

**Review by RickStormBringer — APPROVED** [review]

File: packages/popup/src/components/DappFooter.js (hunk 3 above)
Comment (angelia-yuqi-personal) [reply packages/popup/src/components/DappFooter.js:63]: user can change tx data with the send UI in wallet but I forget the dapp send tx, I will add a task in Clickup to do the dapp send tx data changing.

Comment (angelia-yuqi-personal) [reply packages/popup/src/components/DappFooter.js:63]: https://app.clickup.com/t/3719212/PORTAL-1441

Comment (angelia-yuqi-personal) [reply packages/popup/src/components/DappFooter.js:63]: done

**Pull request merged**